src/sentry/event_manager.py

```diff
@@ -178,6 +179,14 @@ def is_sample_event(job):
     return get_tag(job["data"], "sample_event") == "yes"
 
 
+def normalized_sdk_tag_from_event(event: Event) -> str:
+    try:
+        return normalize_sdk_tag((event.data.get("sdk") or {}).get("name") or "other")
+    except Exception:
+        logger.warning("failed to get SDK name", exc_info=True)
```

Comment:
_If_ we want to keep track of often this fails we should add a DD metric.

Reply:
I don't think we want to. 

The change from `event.data.get("sdk", {}).get("name", "other")` to  `(event.data.get("sdk") or {}).get("name") or "other"` already takes care of the `{"sdk": {"name": null, ...}` situation, so this `try/except` is really just a defense in depth, not expected to get hit.